### ISSUE TITLE
Cannot render multiple images in a pptx table

### ISSUE THREAD

**Issue by illianechesa** [jsreport/jsreport#1036]
I found out that it is currently not possible to use a PPTX image inside multiple rows in a PPTX table. 

For example:
<img width="1271" alt="image" src="https://user-images.githubusercontent.com/47228587/222177652-0088196f-f7b1-4220-84e5-2355491545b3.png">

Here I would expect the image in the middle to repeat in every row. Also the same for the last image (3rd column). However, I cannot achieve this behaviour. 

This is the result:
![image](https://user-images.githubusercontent.com/47228587/222178015-eb9ff11a-49ee-470b-ab49-4fae9b43c741.png)

**Comment by bjrmatos** [comment]
yes, it is currently not supported, the images need to be treated a bit differently in the loop. can you provide the template? it will be easier for us to make this work if you can provide the same template and data, basically an example we can use.

we are currently working on other things related to the docx, so it may take a bit to start the work on this, FYI.

**Comment by illianechesa** [comment]
As far as I saw, it is more of a problem with PowerPoint itself. It does treat images and tables as independent objects, so you cannot really bind an image to a specific cell. The same happens with lists. Here is the template: https://playground.jsreport.net/w/illianechesa/fUhgg9Mv

**Comment by bjrmatos** [comment]
good point, we will check it and see what we can do about it, thanks

**Comment by bjrmatos** [comment]
we have reviewed this, however because the limitation of the pptx format, this is not possible the image will always be floating and generating them from a loop has a big problem, we don't have a reliable way to position the images that are generated from the loop, doing it requires knowledge around the actual dimensions of the slide and rest of content in it, and such information is only available at runtime when you open a pptx file in a editor app (like PowerPoint)

we don't have a solution for this, but we will keep this open and review it again in the future if some new arguments pops up

**Comment by illianechesa** [comment]
> we have reviewed this, however because the limitation of the pptx format, this is not possible the image will always be floating and generating them from a loop has a big problem, we don't have a reliable way to position the images that are generated from the loop, doing it requires knowledge around the actual dimensions of the slide and rest of content in it, and such information is only available at runtime when you open a pptx file in a editor app (like PowerPoint)
> 
> we don't have a solution for this, but we will keep this open and review it again in the future if some new arguments pops up

as a workaround, there is the option to bind an image to the background of a cell, I think this could be a partial "solution"

**Comment by bjrmatos** [comment]
@illianechesa we also considered that approach but we are not yet convinced that it should work like this, we decided to wait for more feature requests/comments before pushing an implementation
